docs/discover/search.asciidoc

```diff
@@ -92,10 +92,10 @@ status:[400 TO 499] AND (extension:php OR extension:html)
 
 
 [[save-open-search]]
-=== Save a search
-A saved search persists your current view of Discover for later retrieval and reuse. You can reload a saved search into Discover, add it to a dashboard, and use it as the basis for a visualization.
+=== Save a Discover Session
+A saved Discover Session persists your current view of Discover for later retrieval and reuse. You can reload a saved session into Discover, add it to a dashboard, and use it as the basis for a visualization.
 
-A saved search includes the query text, filters, and optionally, the time filter. A saved search also includes the selected columns in the document table, the sort order, and the current index pattern.
+A saved Discover Session includes the query text, filters, and optionally, the time filter. A saved Discover Session also includes the selected columns in the document table, the sort order, and the current index pattern.
```

Comment:
```suggestion
A Discover Session includes the query text, filters, and optionally, the time filter. A Discover Session also includes the selected columns in the document table, the sort order, and the current {data-source}.
```
Suggest dropping "saved" for these instances too. Also, unrelated to our work here, but I noticed these docs still refer to "index pattern", so maybe we should update them to "{data-source}" instead?

On a side note, we repeat "Discover Session" a lot in our docs, which reads a bit oddly IMO, but that's a separate issue since it was already written this way.